### PR TITLE
Create ko.rs

### PR DESCRIPTION
**Pull request by kordood**
Add UI Language Korean
Signed-off-by: kordood <gigacms@gmail.com>

### CODE REVIEW

**Comment by rustdesk** [comment]
Thanks, you need to modify lang.rs also.

**Comment by kordood** [comment]
Great, I updated the lang.rs.

File: src/lang.rs [hunk 1]
```diff
@@ -90,6 +92,7 @@ pub fn translate_locale(name: String, locale: &str) -> String {
         "vn" => vn::T.deref(),
         "pl" => pl::T.deref(),
         "ja" => ja::T.deref(),
+        "ja" => ko::T.deref(),
```
Comment (rustdesk) [inline src/lang.rs]: Wrong here, :)

**Review by rustdesk — CHANGES_REQUESTED** [review]

File: src/lang.rs (hunk 1 above)
Comment (kordood) [reply src/lang.rs]: Oops! Thank you for finding it :D

**Pull request merged**